app/controllers/controlled_work_document_selections_controller.rb

```diff
@@ -42,8 +42,11 @@ def handle_download
     track_page_view(page: "download_#{@form.form_type}#{'_welsh' if @form.language == 'welsh'}")
     JourneyLogUpdateService.call(assessment_id, cookies, form_downloaded: true)
     ControlledWorkDocumentPopulationService.call(session_data, @form) do |file|
+      prefix = "#{I18n.t('generic.welsh_in_welsh')} " if @form.language == "welsh"
+      form_name = I18n.t("checks.end_of_journey.form_types.#{@form.form_type}")
+      timestamp = Time.zone.now.in_time_zone("London").strftime("%Y-%m-%d %H.%M.%S")
       send_data file,
-                filename: "controlled-work-form-#{assessment_code}.pdf",
+                filename: "#{prefix}#{form_name} #{timestamp}.pdf",
```

Comment:
should there be a hyphen between the name and timestamp? I know file systems can handle spaces but it jars for me.

Reply:
ah i see the space is consistent with how the rast of the name appears. ignore me.